src/WebJobs.Script.WebHost/Controllers/InstanceController.cs

```diff
@@ -71,5 +71,23 @@ public IActionResult GetInstanceInfo()
         {
             return Ok(_instanceManager.GetInstanceInfo());
         }
+
+        [HttpPost]
+        [Route("admin/instance/disable")]
+        [Authorize(Policy = PolicyNames.AdminAuthLevel)]
+        public async Task<IActionResult> Disable([FromServices] IScriptHostManager hostManager)
+        {
+            if (_environment.IsLinuxContainerEnvironment())
```

Comment:
Aren't all the APIs for this controller specific to LinuxContainers?

Reply:
Yes. Just making it explicit.

Reply:
Then I'd either remove this specific check because it adds confusion, or make it a controller level filter that applies to all actions.